nix/seabug.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, inputs, ... }:
+
+let
+  cfg = config.seabug;
+in
+{
+  options.seabug = {
```

Comment:
should be under `services`, eg. `options.services.seabug`

Reply:
Yeah probably under `services` makes much more sense, not sure why I've put at root level. I'll do it.